src/v2/Components/v2/ArtworkFilter/ArtworkFilters/AttributionClassFilter.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { Checkbox, Flex, Toggle } from "@artsy/palette"
+import { useArtworkFilterContext } from "../ArtworkFilterContext"
+import { OptionText } from "./OptionText"
+
+const checkboxValues = [
+  {
+    name: "Unique",
+    value: "unique",
+  },
+  {
+    name: "Limited Edition",
+    value: "limited edition",
+  },
+  {
+    name: "Open Edition",
+    value: "open edition",
+  },
+  {
+    name: "Unknown Edition",
+    value: "unknown edition",
+  },
+]
+
+export const AttributionClassFilter: React.FC = () => {
+  const filterContext = useArtworkFilterContext()
+
+  const toggleSelection = (selected, name) => {
+    let attributions = filterContext
+      .currentlySelectedFilters()
+      .attributionClass.slice()
+
+    selected
+      ? attributions.push(name)
+      : (attributions = attributions.filter(item => item !== name))
+
+    filterContext.setFilter("attributionClass", attributions)
+  }
+
+  return (
+    <Toggle label="Classification" expanded>
+      <Flex flexDirection="column">
+        {checkboxValues.map(({ name, value }, index) => {
+          const props = {
+            key: index,
+            onSelect: selected => {
+              toggleSelection(selected, value)
+            },
+            selected: filterContext
+              .currentlySelectedFilters()
+              .attributionClass.includes(value),
+          }
```

Comment:
`non-blocking` what's with putting these props in an object vs just placing them onto the component?

Reply:
idk, it just feels cleaner to have all the 'setup' cruft/massaging separate from the final `render` block (which then just spreads in those props, or whatever).

its been a long time since I've PR'ed anything in the Force front-end 😅